### PR TITLE
Local installation of swagger-preview

### PR DESCRIPTION
**Pull request by nisalVD**
Hi,

heres a few things i did in this PR

- added a package.json to the package, and mapped out `swagger-ui-watcher` locally instead of a globally installed node module. 

- added documentation for lazy.nvim

- fixed a typo in documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,14 +10,25 @@ A neovim plugin to live-preview Swagger files.
 
 This plugin requires [swagger-ui-watcher](https://github.com/moon0326/swagger-ui-watcher) to be installed. The default `packer.nvim` configuration will automatically install it globally.
 
+### lazy.nvim
+```lua
+
+    return {
+      "nisalVD/swagger-preview.nvim",
```
Comment (vinnymeller) [inline README.md]:
```suggestion
      "vinnymeller/swagger-preview.nvim",
```

File: README.md [hunk 2]
```diff
@@ -10,14 +10,25 @@ A neovim plugin to live-preview Swagger files.
...
+      cmd = { "SwaggerPreview", "SwaggerStop", "SwaggerPreviewStop" },
```
Comment (vinnymeller) [inline README.md]: should these line up with the commands provided by the plugin?

```suggestion
      cmd = { "SwaggerPreview", "SwaggerPreviewStop", "SwaggerPreviewToggle" },
```

**Review by vinnymeller — COMMENTED** [review]
couple small things otherwise looks good / seems to be working

**Comment by nisalVD** [comment]
Hey thanks for that, added your changes

added a `.stylua.toml` to keep the formatting consistent, happy to change that to whatever you prefer for now i just used whats used in telescope.nvim

**Pull request merged**